src/Donation/TransactionSubscriber.php

```diff
@@ -81,7 +81,13 @@ public function onPayboxIpnResponse(PayboxResponseEvent $event): void
 
         if ($transaction->isSuccessful()) {
             if ($donation->isMembership()) {
-                if ($adherent = $donation->getDonator()->getAdherent()) {
+                $donator = $donation->getDonator();
+                if ($adherent = $donator->getAdherent()) {
+                    $membershipDonations = $donator->getMembershipDonations();
+                    if ($membershipDonations->count() > 1) {
+                        $this->membershipRequestHandler->finishRenaissanceReAdhesion($adherent);
+                    }
+
                     $this->membershipRequestHandler->finishRenaissanceAdhesion($adherent);
```

Comment:
the adherent will receive two emails here, 1 from finishRenaissanceReAdhesion and another from finishRenaissanceAdhesion, normal?

Reply:
no, missing `else` condition